source/main.cpp

```diff
@@ -38,7 +38,7 @@ DigitalOut led_red(LED1);
 DigitalOut led_green(LED2);
 DigitalOut led_blue(LED3);
 
-Serial shared_pc(USBTX, USBRX);
+Serial shared_pc(USBTX, USBRX, 115200);
```

Comment:
Please make this commit add SHARED_SERIAL_BAUD to all targets. This commit should come before the adding of new targets commits, so that each new target added will work in the commit it is added.